.github/workflows/ui-tests-critical.yml

```diff
@@ -54,20 +54,16 @@ jobs:
           - runs-on: macos-13
             xcode: "14.3.1"
             device: "iPhone 14"
-            os-version: "16.4"
-            force-sim-runtime: true
 
           # macos-14 iOS 17 not included due to the XCUIServerNotFound errors causing flaky tests
 
           - runs-on: macos-15
             xcode: "16.2"
             device: "iPhone 16"
-            os-version: "18.2"
```

Comment:
`h`: This removes testing for different OS versions!

Reply:
I added a comment to clarify that it doesn't.